docs/templating/render_content/render_content_in_php.md

```diff
@@ -0,0 +1,46 @@
+---
+description: Render Content item from PHP and get the resulting HTML as a string.
+---
+
+# Render content in PHP
+
+While in PHP, you may need to render the view of a Content item (for further treatment like PDF conversion, because you're not in an HTML context, etc.).
+
+!!! caution
+
+    Avoid using this in a controller as much as possible.
+    You can access directly to a view via the route `/view/content/{contentId}/{viewType}[/{location}]`. For example, on a fresh installation, you can access `/view/content/52/line` which will return a small piece of HTML with a link to the content that could be used in Ajax.
+    If you need a controller (to have additional information available in the template, or to manipulate the `Response` object before returning it), a controller defined in a [view configuration](template_configuration.md) as shown in [Controllers](controllers.md) is a better practice.
+
+The following example is a command outputting the render of a content for a view type in the terminal.
+It works only if the view doesn't refer to the HTTP request.
+It works with views like the `line` or `embed` ones from default installation.
```

Comment:
```suggestion
It is compatibile with the default installation views such as `line` or `embed`.
```

Reply:
- `Use 'it's' instead of 'It is'.  Microsoft.Contractions`
- "compatibile"
Fixed in c74edc43dcb1f9d68273e606cd8a827e41c418d4